experimental/sdg-static/stories.html

```diff
@@ -141,71 +210,200 @@
         font-weight: bold;
         font-size: 1rem;
       }
-      .vertical-line {
-        border-left: 2px solid grey;
-        padding-left: 20px;
-      }
       /** Datacommons style overrides */
       .chart-container {
         height: unset; /** Prevents map chart overflow */
       }
     </style>
   </head>
   <body>
-    <div class="container">
+    <div class="container logo">
       <div class="logo-banner">
-        <img 
-          src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4d/Logo_of_the_United_Nations.svg/800px-Logo_of_the_United_Nations.svg.png"
+        <a href="https://sdgs.un.org/" target="_blank">
+          <img class="left"
+            src="https://sdgs.un.org/themes/custom/porto/assets/images/logo-en.svg"
+          />
+        </a>
+        <a class="right" href="https://datacommons.org" target="_blank">
+          Data<br />Commons
+        </a>
+      </div>
+      <div class="logo-secondary">
+        <a href="https://sdgs.un.org/goals" target="_blank">
+          <img 
+          src="https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/08/E_SDG_logo_without_UN_emblem_horizontal_RGB-1200x219.png"
         />
-        <div class="vertical-line dc-logo">Data<br />Commons</div>
+      </a>
       </div>
     </div>
   
-    <div class="main-title">Sustainable Development Goals</div>
-      <nav class="dc-navbar navbar navbar-expand-lg navbar-light bg-light">
-      </nav>
-      <nav class="dc-subnav">
-        <div class="dc-subnav-item"><a href="/"><span class="dc-subnav-link selected">Global Overview</span></a></div>
-        <div class="dc-subnav-item"><a href="/"><span class="dc-subnav-link">Country View</span></span></a></div>
-      </nav>
+    <div class="main container">
+      <div class="main-title">Sustainable Development Goals</div>
+      <p class="main-text mb-4">
+        Discover the progress made towards the United Nations Sustainable Development Goals (SDGs). Access accurate and up-to-date information, interactive visualizations, and insightful analysis, empowering you to drive positive change towards a sustainable future for all.
+      </p>
+      <p class="main-subtext">Powered by Google <a href="https://datacommons.org" target="_blank">Data Commons</a></p>
+    </div>
+    <nav class="dc-subnav">
+      <div class="dc-subnav-item"><a href="/"><span class="dc-subnav-link selected">Global Overview</span></a></div>
+      <div class="dc-subnav-item"><a href="#"><span class="dc-subnav-link">Country View</span></span></a></div>
+    </nav>
 
 
     <div class="container">
 
       <div class="row">
-
-
-
         <!-- Sidebar -->
-        <div class="col-2">
+        <div class="col-3">
           <div
             class="sticky-top pt-4 pr-4 sidebar-container"
             id="sidebar-container"
           >
-            <div class="sidebar-link" onclick="showSdgContainer(event, 'sdg1')">
+            <div class="sidebar-link selected" data-target="1">
```

Comment:
this seems simpler! I notice same patter when using bootstrap widgets. We can try to accommodate even more.